README.md

```diff
@@ -27,8 +27,8 @@ exampleModule = unsafePartial $ codegenModule "Data.Maybe" do
         , dataCtor "Just" [ typeVar "a" ]
         ]
 
-    , declDerive Nothing []
-        (typeApp "Functor" [ typeCtor "Maybe" ])
+    , declDerive Nothing [] "Functor"
+        [ typeApp (typeCtor "Maybe") [] ]
```

Comment:
I don't think this needs a typeApp
https://github.com/natefaubion/purescript-tidy-codegen/blob/main/test/snapshots/CodegenReadme.purs#L23

Reply:
Sounds good, I just started with this and noticed the example was failing :)

Reply:
Yeah I guess I fixed it in the snapshot and then forgot to update the README 😛